1209/DC21/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Dual Charger
+owner: FPGA-Computer
```

Comment:
Thanks for your pull request!

Please read carefully the guide at http://pid.codes/howto; specifically, you need to create an organisation page for your organisation in addition to the PID page. Please also squash down your changes into a single commit.